package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "dependencies": {
+    "eslint": "^5.8.0",
```

Comment:
Eslint instalation isn't complete

Reply:
Check this guide https://eslint.org/docs/user-guide/getting-started .
You missed .eslintrc and eslint-plugin-import